test/inductor/test_aot_inductor.py

```diff
@@ -143,10 +143,13 @@ def check_model(
     with torch.no_grad(), config.patch(
         "aot_inductor.abi_compatible", self.abi_compatible
     ):
+        torch.manual_seed(0)
         model = model.to(self.device)
         ref_model = copy.deepcopy(model)
         ref_inputs = copy.deepcopy(example_inputs)
         expected = ref_model(*ref_inputs)
+
+        torch.manual_seed(0)
```

Comment:
Wondering why do we need the second call to ```manual_seed```?

Reply:
Because we need to compare the results of eager and compiled, resetting seed makes sure both will generate the same random numbers.

Reply:
Hmm, we actually generate example_inputs before invoking these two manual_seed, i.e. example inputs are passed into the ```check_model``` function. Is there any hidden state that required random numbers?

Reply:
Yeah, in the new test I added.